Translate Jester (10057) (#485)
Giratena/Succubus-Duel-English-Translation: CSV/Language/English/Card/10057.txt

default,
道化師,Jester
EU57_paizuriAnime,
EU57_bustShakeAnime,
EU57_footShakeAnime,
　道化の姿をしたモンスター。つかみどころのない態度であちこち動き回り、気が付けばこちらのそばにいてふところに潜り込んでいる。,A monster who has taken the form of a jester.  Her elusive demeanor and unpredictable movements make her a hard target to keep track of; and before you realize it; she'll have slipped into your space; taking her place right beside you.

> \nどこか憎めない愛嬌のある様子に油断すれば、その時点で彼女の手の平の上であり、主人であるはずのものがかえって召使いに操られ、いつの間にか言われるがまま精も何もかも彼女に搾り取られてしまっているだろう。,As soon as you let your guard down around her seemingly lovable and charming demeanor; you're already in the palm of her hand. Her role of your servant will quickly turn into your mistress; and before you realize it; she'll have milked you of every last drop of cum; and everything you have; all as you obey her every word.

Reviewer: Double 'before you realize' with last sentence. Maybe something like this?

```suggestion
\nどこか憎めない愛嬌のある様子に油断すれば、その時点で彼女の手の平の上であり、主人であるはずのものがかえって召使いに操られ、いつの間にか言われるがまま精も何もかも彼女に搾り取られてしまっているだろう。,As soon as you let your guard down around her seemingly lovable and charming demeanor; you're already in the palm of her hand. Her role as your servant will quickly turn into your mistress; and she'll have milked you of everything you have—your cum, your energy, and your soul—all as you obey her every word.
```

Author: I think this works! Happy to put it in ^^

\n　道化として数々の「芸」を身に着けている彼女は、手コキも足コキもパイズリも得意である。ご主人の好みに合わせて、胸での奉仕を望まれれば弾力たっぷりの胸で愛撫し、足での奉仕を望まれれば絶妙な力加減でぐりぐりと踏みしごいてくれる。のちのことを考えないのであれば、頭を空っぽにし彼女に楽しませてもらうことはとても幸福なことかもしれない。,As a jester who has mastered numerous "performances;" she excels at handjobs; footjobs; and paizuri. She'll cater to her master's preferences—if you desire service from her breasts; she'll caress you with her large; bouncy boobs; and if you want service from her feet; she'll stroke and squeeze your cock between her arches with exquisite pressure control. Should you forget about the situation at hand; you may find yourself trapped in pure bliss; head empty of all thoughts.
[1].このカードは「隠密」「弱点付与」を持つ。,[1]. This card has "Stealth" and "Weakness Assignment.
[2].このカードが通常召喚に成功したとき、LPを800払xcv って発動する。相手フィールド上の最もレベルが低いモンスター（このカードと同名のカードは除く）とこのカードのコントロールを入れ替える。その後、そのモンスターに「攻撃不能（1ターン）」を付与する。,[2]. When this card is successfully Normal Summoned; pay 800 LP to activate. Switch control of this card with the lowest level monster on your opponent's field (excluding cards with the same name as this card). Then; inflict "Cannot Attack (1 Turn)" on that monster.

> [3].このカードのコントローラーが元々の持ち主と違う場合、このカードは相手キャラクターに攻撃できず、自分キャラクターに対してのみ攻撃できる。このときこのカードは反撃されない。,[3]. If this card's controller differs from its original owner; this card cannot attack opponent characters and can only attack your own characters. This card does not receive counterattacks when doing so.

Reviewer: I feel like this could use some rewording. Not sure about the first bit, but I do feel like it needs a change in one way or another

```suggestion
[3].このカードのコントローラーが元々の持ち主と違う場合、このカードは相手キャラクターに攻撃できず、自分キャラクターに対してのみ攻撃できる。このときこのカードは反撃されない。,[3]. If this card's controller doesn't it; it cannot attack opponent characters and can only attack allies. It does not receive counterattacks when doing so.
```

わわ、主様、そのカード上級モンスターですか？\nううっ、道化めをコストにしてしまうだなんて、\nひどいですよ、主様ぁ～。,Wah! Master; is that card an advanced monster? Awh man... You're planning on sacrifcing me to summon it?—That's so mean; Master~
道化めは主様のこと、とっても気持ちよくできますよ？\nだから、わたしをコストになんてしないでください。\nおねがいです～❤主様ぁ❤,Don't you realize how good I can make you feel? Oh; please don't sacrifice me; Master. I'm begging you~❤
例えばほら、このおっぱいを使って・・・、主様の手をむにゅり❤\nその上級モンスターごと挟んでしまいます❤\nどうですかぁ、道化めのおっぱい柔らかいですかぁ～❤,For example; just let me…❤ Wrap my boobs around your hand and trap that card you have; squish~❤ Tell me Master; can you feel how soft and lovely my big boobs are?~❤
ううっ・・・❤,Uuu…❤
ねえ～、主様ぁ❤手の平が気持ちよくて、もうモンスターを出そうとか\nどっちでもよくなってきちゃいましたよね～❤\nあ～ん❤命じてくださればわたくしたっぷりご奉仕できるのにな～❤,Hey~ Master~❤ Your hand feels so good between my tits now you can't even think of summoning anymore; right?~❤ Ahhh~❤ All you have to do is give the order; and I'd service you properly~❤
おねがいです、主様ぁ～❤\n後生ですから道化めをお助けくださいまし～。\nおっぱいでご奉仕させてくださいよ～❤主様ぁ❤あ～ん❤,Please Master~❤ I'm begging you; please spare this jester~ Let me service you with my breasts~❤ Master~❤ Ahh~❤

> 上級召喚をやめてしまった,Stop the advanced summon

Reviewer 2: Suggestion:
Stop -> Halt
or
Stop -> Cancel

Author: I think it's fine as is ^^

いいんですよ❤わたしは仲間なのですから❤\nシコシコして、頭真っ白になって、\n甘えてしまってもよいのです❤,It's alright~❤ I'm your ally after all.❤ Go ahead and stroke yourself; let your mind go completely blank; and just indulge in my attention.❤
道化めの声だけを聞いて、おちんちんをお扱きください、主様ぁ❤\nそうそう上手ですよ～、さすがお仲間さまです❤,Just listen to this jester's voice and stroke your cock; Master~❤ Yes; that's it~ You're so skilled; as expected of my dear friend!❤
ほら、ほら、しこしこしこしこ❤\n道化めにお仲間さまのエナジーをお恵みくださいませ～❤\nふぅーーー・・・❤,There; there; stroke; stroke; stroke; stroke~❤ Bless this jester with your energy; dear friend~❤ Haah...❤
きゃは❤どくどくどくっと、濃ゆいのが出ております❤\nお仲間さまの子種が道化めの手の平の上に、こんなにたくさん❤,Kyaha!❤ It's spurting out in thick; heavy pulses!❤ So much of my dear friend's seed; all over this jester's palm~❤
道化めのサポートはいかがだったでしょうか？\n存分に楽しませることができたのなら、道化冥利に尽きるというもの♪\nくす、気持ちよさそうな顔ですね、お仲間さま❤,I did a good job supporting you; right? If I managed to entertain you thoroughly; then I couldn't be happier as a jester!♪ Kehehe; what a happy face you're making; my dear friend~❤

> おやおや、わたしのような雑魚モンスター相手にオナニーですか？\nくす、効果も不発でほとんど戦闘能力もないというのに、\n道化めの体はそんなにエッチだったでしょうか❤,Oh my; are you masturbating to a weak monster like me? Kehehe; even though my skills barely work and I have almost no combat ability; am I really that sexy of a jester?❤

Reviewer 2: ```suggestion
おやおや、わたしのような雑魚モンスター相手にオナニーですか？\nくす、効果も不発でほとんど戦闘能力もないというのに、\n道化めの体はそんなにエッチだったでしょうか❤,Oh my; are you masturbating to a weak monster like me? Kehehe; even though my skills barely do anything and I have almost no combat ability to speak of; am I really that sexy of a jester?❤
```

Author: ```suggestion
おやおや、わたしのような雑魚モンスター相手にオナニーですか？\nくす、効果も不発でほとんど戦闘能力もないというのに、\n道化めの体はそんなにエッチだったでしょうか❤,Oh my; are you masturbating to a weak monster like me? Kehehe; even though my skills barely do anything and I have almost no combat ability to speak of; do you still find my body sexy?❤
```

あん、どろどろの精液がたっぷり谷間に出ていますね～❤\n道化のパイズリはいかがでしたか？\n腰が抜けるほど気持ちよかったでしょうか。くすくす❤,Mmm; so much thick; gooey cum all over my cleavage~❤ Did you enjoy playing with my funbags? Could you feel your legs give out as I rolled them around your cock? Kehehe~❤
あは、手で扱かれてびゅるるぅっと出してしまいました❤\n指ぬき手袋でしこしこされるのは気持ちよかったですか❤\n道化は喜んでもらえてうれしいです♪,Aha! I stroked you with my hands and you came spurting out~❤ Did it feel good being jerked off by my fingerless gloves?❤ This jester is so happy you enjoyed it♪
あん、足裏で勢いよく射精しています❤\n足ですりすりと擦られて我慢できませんでしたか？\n射精させるのは玉乗りするより簡単ですね～❤くすくす❤,Mmm; you're ejaculating so vigorously against the soles of my foot~❤ Couldn't hold back from being rubbed by my arch? Making you cum is easier than riding a ball~ Kehehe~❤
主様ぁ、どうされましたか？もう体が動かないのですか？\nおよよよ、道化はやりすぎてしまったようです❤,Master~; what's wrong? Can't move your body anymore? Oh my; it seems this jester went too far~❤
これぐらいなら耐えられると思いましたのに、\n気持ちよくしすぎてしまって、ごめんなさいです、主様ぁ❤\nぷっ、くすくすくす❤,I thought you could endure this much; but I made you feel too good; sorry about that; Master~❤ Pfft; *Kehehe Kehehe*❤

> ふふ、何やってるのかしら、{主人公}。\n道化師は敵のモンスターを奪う強力なモンスターだけれど、そっちに\nコントロールが移ってからはちゃんとあなたの命令を聞くはずよ？,Fufu; what are you doing; {主人公}? The jester is a powerful monster that steals enemy monsters; but once control shifts to your side; she should obey your commands properly; right?

Reviewer 2: ```suggestion
ふふ、何やってるのかしら、{主人公}。\n道化師は敵のモンスターを奪う強力なモンスターだけれど、そっちに\nコントロールが移ってからはちゃんとあなたの命令を聞くはずよ？,Fufu; what are you doing; {主人公}? The jester may be a powerful monster that steals away enemy monsters; but once control shifts to your side; she ought to obey your commands properly; right?
```

Author: ```suggestion
ふふ、何やってるのかしら、{主人公}。\n道化師は敵のモンスターを奪う強力なモンスターだけれど、そっちに\nコントロールが移ってからはちゃんとあなたの命令を聞くはずよ？,Fufu; what are you doing; {主人公}? The jester may be a powerful monster that steals away enemy monsters; but don't worry; once she's on your field; I'm sure she'll do whatever you tell her to~
```

自分から道化師にエッチなことを命じて、勝手に負けて、\nカードマスター失格ね❤,You gave the jester lewd commands yourself and lost on your own accord—disqualified as a cardmaster~❤
あなたが負けたらみんな吸い殺されちゃうかもしれないのに、\nふふふ、わたしから溢れているサキュバスの淫気に当てられて、\n理性を失っちゃったのかしら❤,If you lose; everyone might get sucked dry; and yet; fufufu; I guess the succubus aura around me caused you to stop thinking~❤

> うう、そうだ、僕は何をやっているのだろう。いまは欲望に流されていい状況じゃない。サキュバスを前にして命がかかっているのだ。,Ugh; that's right; what am I doing? This isn't a situation where I can give in to desire. Lives are at stake in front of this succubus.

Reviewer 2: ```suggestion
うう、そうだ、僕は何をやっているのだろう。いまは欲望に流されていい状況じゃない。サキュバスを前にして命がかかっているのだ。,Ugh; that's right; what am I doing? This isn't a situation where I should be giving in to temptation. Lives are at stake in front of this succubus.
```

> だが、エルルさんがサキュバス化してから、狭い地下通路の中に甘ったるい空気が漂っている。その匂いを嗅いでいると体がむらむらして仕方がなくなる。,But ever since Elulu turned into a succubus; a sickeningly sweet atmosphere has been drifting through this narrow underground passage. Breathing it in all this time had made me so horny I couldn't think straight.

Reviewer 2: Weren't these sections supposed to be in past tense?

Author: Yes. I will fix them thank you!

```suggestion
だが、エルルさんがサキュバス化してから、狭い地下通路の中に甘ったるい空気が漂っている。その匂いを嗅いでいると体がむらむらして仕方がなくなる。,But ever since Elulu turned into a succubus; a sickeningly sweet atmosphere was now drifting through the narrow underground passage. I was begining to feel so horny from breathing it in I couldn't think straight.
```

> そのうえ、自分のそばにやってきた、この怪しい女の子が、目の前でぴょんぴょんと跳ねて誘惑してくる。密着しそうな距離で、わざとらしくお尻や胸を揺らして誘惑してきて、それで・・・。,On top of that; this suspicious girl who came close to me is bouncing up and down in front of me; tempting me. At such close proximity she's deliberately swaying her ass and breasts to seduce me; and because of that...

Reviewer 2: ```suggestion
そのうえ、自分のそばにやってきた、この怪しい女の子が、目の前でぴょんぴょんと跳ねて誘惑してくる。密着しそうな距離で、わざとらしくお尻や胸を揺らして誘惑してきて、それで・・・。,On top of that; this suspicious girl who came close to me is bouncing up and down in front of me; tempting me. She's deliberately swaying her ass and breasts at such close proximity to seduce me; and because of that...
```

Author: ```suggestion
そのうえ、自分のそばにやってきた、この怪しい女の子が、目の前でぴょんぴょんと跳ねて誘惑してくる。密着しそうな距離で、わざとらしくお尻や胸を揺らして誘惑してきて、それで・・・。,On top of that; this suspicious girl who came close to me is bouncing up and down in front of me; tempting me. Deliberately swaying her ass and breasts at in front of my very eyes; and...
```

リズムを取りまして、すりすりすりすり♪\nごしごしごしごし♪\nにひひ、いかがですか観客さまぁ❤おちんちん気持ちいいですか～❤,Keeping rhythm; rubbing; rubbing; rubbing!♪ Grinding; grinding; grinding!♪ Hehe; how is it; dear audience member?❤ Does your cock feel good~?❤
おっと、口からお返事を聞くまでもなく、\nおちんちんがびくびくと返事をしていますね～❤\nわたくしの芸を、喜んでいただけているようでなによりです♪,Oh my; I don't even need to wait for a verbal answer—your cock is twitching its response~❤ I'm so glad you're enjoying my performance!♪
観客さまのおひねりを、さあさあ、道化めの足にお出しください❤\n命を溶かして、白くて濃ゆいのを、\nブーツに踏まれて出してくださいまし❤,Now then; give your tip to this jester's feet; dear audience member.❤ Melt your life away and shoot that thick; white load all over my boots.❤
はひっ、あっ、ぐりぐりこすられてっ❤あああっ❤,Hahh!❤ Ah!❤ Grinding against me like that!❤ Aaaah!❤
フィナーレは早く振動させながらこすりあげますね❤\n我慢などで出来ぬよう、搾りだしますゆえ❤\nどうぞ、ほら、びゅっ、びゅっ、びゅっ～❤,For the finale; I'll rub you quickly with rapid vibrations.❤ I'll squeeze every last drop out so you can't hold back anymore.❤ Come on now; hora; spurt; spurt; spurt~❤
にひひ❤たくさん出しましたね、観客さま❤\nおや、いっぱい出したせいで、体がだいぶやつれてきているご様子❤\nエナジーが失われてだいぶ苦しいのではないですか？,Ehehehe!❤ You came so much; dear audience member.❤ Oh my; after releasing all that; your body looks quite drained.❤ You must be suffering now that your energy leaked out; right?

> ですが、苦しいのとは別に、頭の奥がじんと痺れたように\n心地よいのですよね～❤サキュバスに吸われるときは\n得てしてそういうもの❤抵抗する気力は快楽に奪われるのです❤,But you're not suffering aren't you? Can you feel your mind getting foggy and sleepy~?❤ That's just how it is when a succubus drains you.❤ The pleasure steals away your will to resist.❤

Reviewer 2: "You must be suffering", followed up with "But you're not suffering"?

Author: Hmm good catch Shrug.

```suggestion
ですが、苦しいのとは別に、頭の奥がじんと痺れたように\n心地よいのですよね～❤サキュバスに吸われるときは\n得てしてそういうもの❤抵抗する気力は快楽に奪われるのです❤,That said; you're not suffering at all are you? Can you feel your mind getting foggy and sleepy~?❤ That's just how it is when a succubus drains you.❤ The pleasure steals away your will to resist.❤
```
